h2o-py/h2o/explanation/_explain.py

```diff
@@ -4,6 +4,7 @@
 import warnings
 from contextlib import contextmanager
 from collections import OrderedDict, Counter, defaultdict
+from h2o.plot.plot_result import decorate_plot_result
```

Comment:
@koniecsveta may I ask you a favour?

- `plot_result.py` -> `_plot_result.py` : we want ot make it clear to users that it's not safe to use those modules directly, and that they can change any time.
- create module `h2o/plot/__init__.py` which exports `decorate_plot_result`
- then, please always import directly from `h2o.plot`.
- extra bonus: also move `h2o/utils/ext_dependencies.py` to `h2o/plot/_matplotlib.py` and also ensure it's imported directly from `h2o.plot`
At the end some files should have sth like:
```python
from h2o.plot import get_matplotlib_pyplot, decorate_plot_result
```
which is better structured with all plot-related functions from the same module.

Reply:
okok 👌  d3db906